docs/pt/docs/advanced/sub-applications.md

```diff
@@ -0,0 +1,73 @@
+# Sub Aplicações - Montagens
+
+Se você precisar ter duas aplicações FastAPI independentes, cada uma com seu próprio OpenAPI e suas próprias interfaces de documentação, você pode ter um aplicativo principal e "montar" uma (ou mais) sub-aplicações.
+
+## Montando uma aplicação **FastAPI**
+
+"Montar" significa adicionar uma aplicação completamente "independente" em um caminho específico, que então se encarrega de lidar com tudo sob esse caminho, com as operações de caminho declaradas nessa sub-aplicação.
+
+### Aplicação de nível superior
+
+Primeiro, crie a aplicação principal, de nível superior, **FastAPI**, e suas *operações de caminho*:
+
+```Python hl_lines="3  6-8"
+{!../../../docs_src/sub_applications/tutorial001.py!}
+```
+
+### Sub-aplicação
+
+Em seguida, crie sua sub-aplicação e suas *operações de caminho*.
```

Comment:
Most translations in pt docs are using operações de rota when translating path operations